zebrad/tests/acceptance.rs

```diff
@@ -20,7 +20,91 @@
 //! If it does not have any IPv4 interfaces, IPv4 localhost is not on `127.0.0.1`,
 //! or you have poor network connectivity,
 //! skip all the network tests by setting the `ZEBRA_SKIP_NETWORK_TESTS` environmental variable.
-
+//!
+//! ## Large/full sync tests
+//!
+//! This file has sync tests that are marked as ignored because they take too much time to run.
+//! Some of them require environment variables or directories to be present.
+//! Here are some examples on how to run each of them:
+//!
+//! ```console
+//! $ cargo test sync_large_checkpoints_mainnet -- --ignored --nocapture
```

Comment:
Consistency: The scripts here are slightly different to what we run in CI.

Reply:
We've had problems when we make small changes to test names or flags, and they get skipped. Because we can't really detect that in CI. That's why the `entrypoint.sh` script uses `--include-ignored` for all tests.

`--ignored` means "only run the test if it is marked `#[ignored]` in Rust".

`--include-ignored` is more reliable, because it means "run the test with this name, it doesn't matter if it is `#[ignored]` or not".

Conrado's point about compile speed is also a good one, but that is just an efficiency thing. It won't give us the wrong test results.